### ISSUE TITLE
DB Not Deployed

### ISSUE THREAD

**Issue by evalaidc** [DavidZafa/DavidZProject2#1]
When you ran node <path to seed.js> did you get any errors? 
There is an unfilled key for Steph Curry. Might get rid of the issue when trying to seed your db. 
Once you get your db set up, you will need to deploy using MLAB and also adjust your db to listen for it. 
https://devcenter.heroku.com/articles/mongolab
If you have any questions about it, let me know.
In your seed database, you also might want to start by clearing the users and all your db information and then adding the basketball players, users, and teams. 
ex:
```
User.find({}).remove(() => {
// Remove any additional stuff, then once removed, you can start by creating a user  and then 
 // nesting teams/players inside or by creating the users and then the players individually, then 
 // choose which players you want added to their teams.
```

Let me know if you have additional questions

**Comment by DavidZafa** [comment]
Hey Eva,

All done!

https://davidzafirovicfullstack.herokuapp.com/login

**Comment by evalaidc** [comment]
@DavidZafa  Hi David, It looks like your players aren't populating the database on your deployed heroku site. 

Also for seeding your database, I meant you could do something like what I wrote below.

```
// erases all models that match Basketball model
Basketball.remove({}).then(function(){
 playerData.forEach(player => {
  Basketball.create({
    info: {
      image: player.image,
      name: player.name,
      position: player.position,
      team: player.team,
      number: player.number,
      points: player.points
    }
  })
  .then(players => {
    players.save(err => console.log(err))
  })
});

```

this way you are creating a Basketball object for each player. 

maybe you should have a Team model where you can push players into the team as they get added. It would probably look a little something like this:
```
const Schema = mongoose.Schema
const Basketball = require('../models/basketball')

const team = new Schema({
  teamRoster: [Basketball]
  teamName: String
})
```
Let me know if you have other questions.

**Comment by DavidZafa** [comment]
Hey Eva,

Actually everything should be working, I tested the deployed app myself. The thing is all the seeded data is case-sensitive so you have to type it in like Lebron James, Kevin Durant, Stephen Curry

**Comment by evalaidc** [comment]
Hi David,

Ah I see now what you mean. The search was kind of misleading to me then. Maybe a good alternative would be to list out the players or post a random sample of some when adding players to the team.

Keep in mind, when someone searching for something and it doesn't exist in the database, they get an internal server error. For a bonus, you could create some type of error handler like an alert that says that they couldn't find anything that matched, or you could adjust your call to recognize parts of the inputted text in your controller.
https://mongoosejs.com/docs/api.html#model_Model.find

**Comment by DavidZafa** [comment]
Yeah, sorry for the confusion. I think I’m just going to go back and create a drop-down in the search to make it more simple and save the trouble of the error.

**Comment by evalaidc** [comment]
No worries, just want your project ready for other potential employers to see. Haha yeah that is a good alternative :)
